sepses/.htaccess

```diff
@@ -1,4 +1,36 @@
 Options +FollowSymLinks
+
 RewriteEngine on
+
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+
 RewriteRule ^$ http://sepses.ifs.tuwien.ac.at [R=302,L]
 RewriteRule ^(.*)$ http://sepses.ifs.tuwien.ac.at/$1 [R=302,L]
```

Comment:
Rules are checked in order, so this wildcard rule will catch everything.  You should move it to the bottom of the file.

Reply:
done.. thankyou

Reply:
Hi I still got a problem when I open this link  http://w3id.org/sepses/vocab/ref/cve/ontology.ttl via protege, the error message is :
sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

but when I directly open this link: http://sepses.ifs.tuwien.ac.at/vocab/ref/cve/ontology.ttl, it works fine.

do you see that this is a problem from w3id?